api/tests/certification/enrolment/unit/domain/services/register-candidate-participation-service_test.js

```diff
@@ -52,7 +53,7 @@ describe('Unit | Application | Service | register-candidate-participation', func
   context('when the candidate is not yet linked to a user', function () {
     it('should link the candidate to the given user', async function () {
       // given
-      const unlinkedCandidate = domainBuilder.certification.enrolment.buildEnrolledCandidate({
+      const unlinkedCandidate = domainBuilder.certification.enrolment.buildCandidate({
         ...candidateData,
         userId: null,
```

Comment:
Histoire de bien voir le couple `userId` et `reconciledAt` quand on parle d'un candidat réconcilié ou non

```suggestion
        userId: null,
        reconciledAt: null,
```

Reply:
J'ai pris en compte !